curriculum/challenges/english/03-front-end-development-libraries/bootstrap/add-font-awesome-icons-to-all-of-our-buttons.md

```diff
@@ -25,28 +25,28 @@ You should add a `<i class="fas fa-info-circle"></i>` within your `info` button
 
 ```js
 assert(
-  $('.btn-info > i').is('.fas.fa-info-circle') ||
-    $('.btn-info > span').is('.fas.fa-info-circle')
+  document.querySelector('.btn-info > i') === document.querySelector('.fas.fa-info-circle') ||
+    document.querySelector('.btn-info > span') === document.querySelector('.fas.fa-info-circle')
```

Comment:
My tests unfortunately shown this always passes. Initially when neither is yet added both sides will be `null`, so assertion passes.

```suggestion
  document.querySelector('.btn-info > i')?.classList?.value === 'fas fa-info-circle' ||
    document.querySelector('.btn-info > span')?.classList?.value === 'fas fa-info-circle'
```